subsys/net/l2/ethernet/eth_stats.h

```diff
@@ -17,9 +17,13 @@ static inline void eth_stats_update_bytes_rx(struct net_if *iface,
 					     u32_t bytes)
 {
 	struct net_stats_eth *stats;
+	const struct ethernet_api *api = ((const struct ethernet_api *)
+		net_if_get_device(iface)->driver_api);
 
-	stats = ((const struct ethernet_api *)
-		 net_if_get_device(iface)->driver_api)->stats;
+	if (!api->get_stats)
```

Comment:
Missing {}

Reply:
Done, updated in dcbadc52a66f1e31e9dbe081fa4aeb2e6c36a727.